client/app/dedicated/server/bandwidth/cancel-private/dedicated-server-bandwidth-vrack-cancel.html

```diff
@@ -0,0 +1,25 @@
+<div data-wizard
```

Comment:
Please don't use `wizard`, we haven't used it for a year now

Reply:
It's not new, it has moved